### PR TITLE
Feature/merges

### LINKED ISSUE #18: [0.5pt] Set up the project with Webpack and Jest- group task
- Set up Gitflow
- Set up webpack
- Set up a JavaScript testing library (Jest)

### LINKED ISSUE #19: [1pt] Choose assets - group task
Choose images, text, icons, fonts.

### PR DESCRIPTION
**Pull request by mershark**
In this project together with my partner @hamidazim321 we created a fully functional meal webpage. Some of the things we implemented in this task are:
- We utilized an external API ["Themealdb"](https://www.themealdb.com/api.php) to fetch data about a topic of interest.
- We created 2-3 user interfaces: home page with "likeable" items and comments popup.
- We employed JavaScript best practices, ES6 syntax, and modules.
- We implemented webpack for bundling and Gitflow for version control.
- We integrated the  ["Involvement API"](https://www.notion.so/microverse/Involvement-API-869e60b5ad104603aa6db59e08150270) to record user interactions (likes and comments).
- We conducted code reviews for each other's work.
- We developed unit tests for the JavaScript app using Jest.

## Changes after the previous review
**We have added all the features required by the reviewer**
- We have separated both counter functions into separate modules
- We have updated the test for the case when there are no meals on the home screen (e.g. the API request does not return the expected data)

### CODE REVIEW

File: src/Modules/__tests__/counter.test.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    commentsCounter(arr2, header);
+    expect(header.textContent).toBe('Comments (0)');
+
+    commentsCounter(arr3, header);
+    expect(header.textContent).toBe('Comments (0)');
+  });
+});
+
+describe('Meals Counter', () => {
+  test('Update Header to have the correct Count of Items', () => {
```
Comment (muneebulrehman) [inline src/Modules/__tests__/counter.test.js]: Kindly also test for the case when there are no meals on the home screen like you tested for the comments. As per the project requirements, tests should cover all the edge cases.

File: src/Modules/counter.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+};
+
+const commentsCounter = (arr, header) => {
+  if (Array.isArray(arr)) {
+    header.textContent = `Comments (${arr.length})`;
+  } else {
+    header.textContent = 'Comments (0)';
+  }
+};
+export { mealCounter, commentsCounter };
```
Comment (muneebulrehman) [inline src/Modules/counter.js]: One of the project requirements is that each counter should be implemented as a separate module. Kindly move each of these counters into a separate file. 
image

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi team, 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Followed template design
- Used involvement API
- Split code in proper modules.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by iambenkis — APPROVED** [review]
Hi @mershark , @muneebulrehman ,

## APPROVED 🟢

## Highlights

- [ ] Innovative idea in a good-looking website ✔️ 
- [ ] good UI design ✔️ 
- [ ] no linter errors ✔️ 
- [ ] correct github flow is used ✔️ 
- [ ] good PR description ✔️ 

Great implementation of all the requirements in this milestone. 👏 

_You have done an amazing job 👍_

Your project is complete!  There is nothing else to say other than... it's time to merge it. :shipit: 

Congratulations! 🎉 

## CHEERS AND HAPPY CODING!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
